lib/grape-swagger/doc_methods/build_model_definition.rb

```diff
@@ -17,6 +17,57 @@ def build(model, properties, required)
           definition
         end
 
+        def parse_params_from_model(parsed_response, model, model_name)
```

Comment:
One of my strong doubts is related to this part of the code. Instead of returning a tuple with a definition and list of attributes, we start to return just a `Hash`. Of course, this because we previously return only the `properties` field, and now we need to return full definition as is.

In my opinion, `grape-swagger-entity` should return the final definition without an addition wrapping ( like we do this in the `build` method ). But I do not insist if it ok for @LeFnord.

Reply:
But if we do that, it result a big breaking change for all parser no ?

Reply:
Right. In my mind, we can declare a wrapper class with a method `to_definition` on the `grape-swagger` side. Parsers will use this class or can use own inherited from this class to declare more complex logic.

For the backward compatibility, we can check is the result of the parser is an array or wrapper object.

Reply:
What do you think @LeFnord ?

Reply:
Moinsen … and sorry for latency …

for this PR I'm fine with the actual solution, but we should change the behaviour as @Bugagazavr mentioned above in another PR to make things clear as a breaking change

Reply:
@LeFnord let me know when you release this feature. I need to release `grape-swagger-entity` with dependency lock to the latest `grape-swagger`.

Reply:
@Bugagazavr … I can not release it, cause `grape-swagger-entity` references a git branch 

maybe you releasing `grape-swagger-entity` and set the grape-swagger dep to 1.4.0, after that I can change it here, mörge it and make a new release 1.4.0

what di you think?

Reply:
Done https://rubygems.org/gems/grape-swagger-entity